src/options.cpp

```diff
@@ -1429,12 +1429,12 @@ void options_manager::init()
 
     mOptionsSort["world_default"]++;
 
-    add( "STATIC_NPC", "world_default", translate_marker( "Static npcs" ),
-        translate_marker( "If true, the game will spawn static NPC at the start of the game, requires world reset." ),
+    add( "STATIC_NPC", "world_default", translate_marker( "Static NPC" ),
```

Comment:
Shouldn't this be "NPCs"

Reply:
Don't NPC's spawn at the refugee center and farm with static NPCs on? Maybe we should change that description.

Reply:
Correct, with Static NPCs enabled, this functionally allows survivors to engage in quest lines to develop the refugee center and ranch and such.

Suggest `Static NPCs` as well, however Static & Dynamic should be changed in the future to better fit what they do. (Static is for more story/long quest lines/developing world, and Dynamic is just randomly generated NPCs that may or may not offer quests -- "Quest NPCs" & "NPC Survivors" ? )